### PR TITLE
Unit 3 objectives

### PR DESCRIPTION
**Pull request by jimflores5**

### CODE REVIEW

File: unit03/lists.rst [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+- Understand that lists, like strings, are indexed collections.
+- Assign a list literal to a variable.
+- Create lists that store multiple values of the same or different data types.
+- Use bracket notation and index values to access or modify one element in a
+  list.
+- Use bracket notation to access or modify a **slice** of multiple elements in
+  a list.
+- Use iteration to loop through the elements within a list.
+- Recognize that lists are **mutable**, and describe what that means.
+- Find the length of an list using the ``len()`` function.
```
Comment (CarlyLanglois) [inline unit03/lists.rst]: a list*

File: unit03/strings.rst [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+- Correctly use the ``+`` and ``*`` operators to concatenate strings.
+- Use template literals to create strings that rely on variable values.
+- Recognize that string methods return new values without changing the
+  original string.
+- Use the ``format()`` method to insert values into a string.
+- Use other common string methods: ``count()``, ``find()``, ``index()``,
+  ``lower()``, ``replace()``, ``strip()``, ``upper()``.
+- Use ``split()`` and ``list()`` to convert a string into a list of string
+  elements.
+- Use boolean methods like ``isalpha()`` and ``islower()`` to check the state
```
Comment (CarlyLanglois) [inline unit03/strings.rst]: isalpha() doesnt look like its in the book so I'd remove this

**Review by CarlyLanglois — APPROVED** [review]

File: unit03/lists.rst (hunk 1 above)
Comment (jimflores5) [reply unit03/lists.rst]: Fixed, thanks!

File: unit03/strings.rst (hunk 2 above)
Comment (jimflores5) [reply unit03/strings.rst]: Removed.

**Pull request merged**